control-plane/csi-driver/src/bin/node/filesystem_ops.rs

```diff
@@ -226,18 +226,17 @@ impl FileSystemOps for Ext4Fs {
         Ok(())
     }
 
-    async fn expand(&self, mount_path: &str) -> Result<(), Error> {
-        let dev_path = get_devicepath(mount_path)
-            .map_err(|err| {
-                format!(
-                    "failed to get dev path for mountpoint {}: {}",
-                    mount_path, err
-                )
-            })?
-            .ok_or(format!(
-                "no underlying device found for mountpoint {}",
-                mount_path
-            ))?;
+    async fn expand(&self, mount_path: &str, dev_path: Option<String>) -> Result<(), Error> {
+        let dev_path = match dev_path {
```

Comment:
Why does this use dev_path and the others seem to use mount_path ?

Reply:
It just does, https://linux.die.net/man/8/resize2fs. Takes the 'device' as the argument.
https://linux.die.net/man/8/xfs_growfs takes the mountpoint as the argument.
https://man7.org/linux/man-pages/man8/btrfs-filesystem.8.html takes the path (see EXAMPLES section).